browser-checklist.md

```diff
@@ -0,0 +1,26 @@
+## Browsers
+
+- [ ] Chrome
+- [ ] Safari
+- [ ] FF
+- [ ] IE10
```

Comment:
I assume un-versioned entries mean latest and greatest. Do you want IE11 included?